website/src/main/tut/docs/users/configuration.md

```diff
@@ -109,4 +111,38 @@ debug.printSymbols = true
 
 To build custom rules see {% doc_ref Rule Authors %}.
 
+## Per source file
+
+It's possible to supress false positives in the source code itself to have a more fine grain on what to report and what not to report. There is two techniques you could use to disable linting: disable/enable until end of file or disable per expression.
+
+### Enable or Disable until the EOF
+
+```scala
+
+// scalafix:off
+foo(null)
+1.asInstanceOf[String]
+// scalafix:on
```

Comment:
I would first show an example without `scalafix:on`. Then show another example how to re-enable.